crates/bevy_ecs/src/world/mod.rs

```diff
@@ -921,7 +921,8 @@ impl World {
 
     /// Spawns a new [`Entity`] with a given [`Bundle`] of [components](`Component`) and returns
     /// a corresponding [`EntityWorldMut`], which can be used to add components to the entity or
-    /// retrieve its id.
+    /// retrieve its id. In case large batches of entities need to be spawned, consider using
+    /// [`World::spawn_batch`] instead.
```

Comment:
Note that this is `World::spawn` not `Commands::spawn`, we still need to document it in the latter